xcm/pallet-xcm/src/lib.rs

```diff
@@ -106,6 +116,13 @@ pub mod pallet {
 			+ GetDispatchInfo
 			+ IsType<<Self as frame_system::Config>::Call>
 			+ Dispatchable<Origin = <Self as Config>::Origin, PostInfo = PostDispatchInfo>;
+
+		/// The maximum number of items that we can store in the version discovery queue.
+		type VersionDiscoveryQueueSize: Get<u32>;
```

Comment:
We should avoid using `Get` where we expect a const, else someone may implement this in a bad way :|

```suggestion
		const VersionDiscoveryQueueSize: u32;
```

Reply:
there are places in substrate we need to do the same

Reply:
this must be passed as a type parameter to a `BoundedVec`. would you expect it to be re-wrapped in a `type_parameter` and passed that way?